### ISSUE TITLE
MapKit.h and UIKit.h

### ISSUE THREAD

**Issue by JHT1958** [Microsoft/WinObjC#330]
Hello I have a compile error after generating with vsimporter on:

 #import <UIKit/UIKit.h>

**Comment by SaeedAkhter-MSFT** [comment]
@JHT1958, thanks for reporting the issue.  Can you copy & paste the error from your build log?

**Comment by JHT1958** [comment]
![capture d ecran 2016-02-06 a 09 12 03](https://cloud.githubusercontent.com/assets/5244281/12865636/c00d3b82-ccb1-11e5-88a3-086631a34b34.jpg)

**Comment by SaeedAkhter-MSFT** [comment]
Sorry, I should have been more clear.  The "build log" can be accessed in two places:

1) In Visual Studio you can find it inside the "Output" window when you set the dropdown "Show output from:" = "Build".
2) Or you can also find the "build log" on the file system.   If your project name was myapp, the build log can be found under myapp.vsimporter\myapp-WinStore10\Debug\myapp.log

**Comment by JHT1958** [comment]
Hello,
 is it you want?


1>------ Début de la génération : Projet : voxmapp (voxmapp\voxmapp), Configuration : Debug Win32 ------
1>  ..\..\proxibuzz\main.m
1>C:\winobjc\samples\voxmapp\proxibuzz/VoxMappAppDelegate.h(9,9): fatal error : 'MapKit/MapKit.h' file not found
========== Génération : 0 a réussi, 1 a échoué, 0 mis à jour, 0 a été ignoré ==========
========== Déploiement : 0 a réussi, 0 a échoué, 0 a été ignoré ==========

**Comment by ilMalvagioDottorProsciutto** [comment]
I've the same problem probably. I remove MapKit from my import (My app can run without it) but still UIKit import gives me errors.
Looks like the linker could not find the "standard" libraries/frameworks. It may be a setting in the project?

**Comment by JHT1958** [comment]
Goodnight For now I stop converting my IOS since it uses MapKit not yet provided for by this development.

I waited a long time this kind of thing, but it's not enough yet finalized.

**Comment by ArnOmsft** [comment]
You won't be able to use MapKit but you can use the projections corresponding to the Windows.Services.Maps and Windows.UI.Xaml.Controls.Maps namespace available in  / include / Platform / Universal Windows / UWP /. The documentation for these namespace is available on msdn:
- https://msdn.microsoft.com/en-us/library/windows/apps/windows.ui.xaml.controls.maps.aspx
- https://msdn.microsoft.com/en-us/library/windows/apps/windows.services.maps.aspx

For UIKit, did you try with our last release and what errors related to UIKit specifically do you get?
Thanks.

**Comment by JHT1958** [comment]
Thank you for this information, but I thought may be a wrong that this project was to convert the iOS applications, also very ambitious project. So rewrite the application using the Windows little interest services.

thanks anyway

**Comment by ArnOmsft** [comment]
[I'm also replying in French below since I saw you're from France and I'm too ;-)]
This project is not designed to allow a super-quick conversion of your iOS app into a Windows app that would work right away, although we saw a few occurrence of that for very simple applications. Usually, once the iOS application is converted into a Visual Studio solution, the Developer will have some work to do to make the app work fine on Windows. The amount of work depends on what iOS framework the app is using (and what coverage we have for these at the time of the conversion), what iOS services are used and what middleware are used as well. For services like MapKit, we technically cannot use the iOS service from a Windows app, so the app has to use the Microsoft equivalent. We're working to provide code samples and documentation showing how to update an app using iOS services to call the Microsoft services instead but we haven't done that for MapKit yet.
I'm going to close this issue, but feel free to open a new one to report any issue you might have when you'll have the chance to try again with our latest release.
Thank you for your contribution.

[Traduction Française]
Ce projet n'a pas vocation a permettre une conversion rapide de votre app iOS en une app Windows qui fonctionnerait immédiatement, bien que nous ayons vu quelques exemple de ca pour des applications très simples. Normalement, une fois l'application iOS convertie en un project Visual Studio, le développer devra apporter des modifications a l'application pour que celle-ci fonctionne parfaitement sur Windows. La quantité de travail va dépendre de quels framework iOS l'application utilise (et de quel niveau de compatibilité le projet offre pour ces framework au moment de la conversion), de quels services Apple (IAP, MapKit, iAds, etc...) l'application utilise, et de quels middleware l'app utilise. Pour les services tel que MapKit, nous ne pouvons techniquement pas appeler ces services depuis une application fonctionnant sur Windows, donc l'application Windows ne peut qu'utiliser le service Microsoft equivalent. Nous prévoyons de poster des exemples de codes montrant comment modifier une application utilisant chaque service iOS pour appeler le service Microsoft equivalent, mais nous n'avons pas encore publier ceci pour MapKit.
Je vais fermer ce ticket, merci d'ouvrir un autre ticket si vous rencontrer des problèmes lorsque vous aura l'opportunité de tester notre dernier SDK.
Merci de votre contribution.
